infrastructure/cdn-in-a-box/optional/grafana/trafficcontrol-scenes/webpack/webpack.config.ts

```diff
@@ -11,23 +11,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/*
- * ⚠️⚠️⚠️ THIS FILE WAS SCAFFOLDED BY `@grafana/create-plugin`. DO NOT EDIT THIS FILE DIRECTLY. ⚠️⚠️⚠️
- *
- * In order to extend the configuration follow the steps in
- * https://grafana.com/developers/plugin-tools/create-a-plugin/extend-a-plugin/extend-configurations#extend-the-webpack-config
- */
+/* eslint-disable */
```

Comment:
You should probably disable a specific rule (with a comment explaining why) and not the entire linter.